xml/dm_registers.xml

```diff
@@ -7,7 +7,13 @@
         the currently selected harts, as defined in \FdmDmcontrolHasel.  Its address will
         not change in the future, because it contains \FdmDmstatusVersion.
 
-        <field name="0" bits="31:24" access="R" reset="0" />
+        <field name="0" bits="31:25" access="R" reset="0" />
+        <field name="ndmresetpending" bits="24" access="R" reset="Preset">
```

Comment:
```suggestion
        <field name="ndmresetpending" bits="24" access="R" reset="-">
```

We've used Preset to mean hardcoded, which this bit isn't quite. (I don't think that's mentioned anywhere, but we can at least be consistent.)